app/api/logging/config.py

```diff
@@ -1,16 +1,27 @@
 import logging
 import sys
 
+import api.logging.audit
 import api.logging.formatters as formatters
 import api.logging.pii as pii
 from api.util.env_config import PydanticBaseEnvConfig
 
 logger = logging.getLogger(__name__)
 
 
+class HumanReadableFormatterConfig(PydanticBaseEnvConfig):
+    message_width: int = formatters.HUMAN_READABLE_FORMATTER_DEFAULT_MESSAGE_WIDTH
+
+
 class LoggingConfig(PydanticBaseEnvConfig):
-    log_format: str = "json"
-    log_level: str = "INFO"
+    format = "json"
+    level = "INFO"
+    enable_audit = True
+    human_readable_formatter = HumanReadableFormatterConfig()
```

Comment:
How would I set something different here?

Is there some way to specify a path in the env var?

Reply:
yeah I gave a commented out example in `local.env`, but you can do
`LOG_HUMAN_READABLE_FORMATTER__MESSAGE_WIDTH=80` to overwrite the values in `HumanReadableFormatterConfig`